client/src/Movies/Imdb.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+//imports a random movie or TV show from IMDB api...
+class Imdb extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movieList: []
+    };
+  }
+  componentDidMount = () => {
+    fetch(
+      "https://www.omdbapi.com/?apikey=8894cd6c&plot=full&i=tt0" +
+        `${Math.floor(Math.random() * (999999 - 100000) + 100000)}`
+    )
+      .then(res => res.json())
+      .then(result => {
+        this.setState({
+          movieList: new Array(result)
+        });
+      });
+  };
+
+  render() {
+    if (!this.state.movieList) {
```

Comment:
```suggestion
    if (this.state.movieList.length === 0) {
```
Even an empty array evaluates to true, I recommend you check to see if the array is populated instead